src/desktop/apps/artist/components/cta.ts

```diff
@@ -0,0 +1,86 @@
+import * as $ from "jquery"
+import { get } from "lodash"
+import { data as sd } from "sharify"
+
+const metaphysics = require("lib/metaphysics.coffee")
+const Artist = require("desktop/models/artist.coffee")
+const ArtistPageCTAView = require("desktop/components/artist_page_cta/view.coffee")
+const mediator = require("desktop/lib/mediator.coffee")
+// Uncomment when we are ready to launch a/b test
+// const split_test = require("desktop/components/split_test.coffee")
+// split_test.view()
+
+export const query = `
+query ArtistCTAQuery($artistID: String!) {
+  artist(id: $artistID) {
+    href
+    name
+    cta_image: image {
+      thumb: resized(width: 150, version: "square") {
+        url
+      }
+    }
+    cta_artists: artists(size: 1) {
+      image {
+        thumb: resized(width: 150, version: "square") {
+          url
+        }
+      }
+      name
+    }
+    artworks(size: 1) {
+      image {
+        cropped(width: 390, height: 644) {
+          url
+        }
+      }
+    }
+  }
+}
+`
+const send = {
+  method: "post",
+  query,
+  variables: { artistID: sd.ARTIST_PAGE_CTA_ARTIST_ID },
+}
+
+export const init = () => {
```

Comment:
I would use a more descriptive function name here since it's being exported. Maybe `setupArtistSignUpModal`